model/base.py

```diff
@@ -1,18 +1,18 @@
 from abc import ABC, abstractmethod
 from color import Color
 from grid.cell import Address, universe_count, universe_size
-from typing import Callable, List, Mapping
-
-SetColorFunc = Callable[[Color], None]
+from typing import List, Mapping, Union
 
 
 class ModelBase(ABC):
     """Abstract base class for simulators."""
 
     @abstractmethod
-    def set(self, addr: Address, color: Color):
+    def set(self, addr: Union[Address, int], color: Color):
```

Comment:
```suggestion
    def set(self, cell: Cell, addr: Address, color: Color):
```

I thought about fixing this by also passing in the `Cell` always. the real sACN model would ignore the cell; the simulator would ignore the address